testing/hierarchies.py

```diff
@@ -1034,10 +1072,10 @@ def compare_class_hierarchy(test, test_root):
 
     # Run some preliminary tests
     exhale_root = get_exhale_root(test)
-    test.assertEqual(len(test_root.class_like), len(exhale_root.class_like))
-    test.assertEqual(len(test_root.enums), len(exhale_root.enums))
-    test.assertEqual(len(test_root.namespaces), len(exhale_root.namespaces))
-    test.assertEqual(len(test_root.unions), len(exhale_root.unions))
+    test.assertEqual(len(test_root.class_like), len(exhale_root.class_like), msg="Classes don't match")
+    test.assertEqual(len(test_root.enums), len(exhale_root.enums), msg="Enums don't match")
+    test.assertEqual(len(test_root.namespaces), len(exhale_root.namespaces), msg="Namespaces don't match")
+    test.assertEqual(len(test_root.unions), len(exhale_root.unions), msg="Unions don't match")
```

Comment:
I added these strings to identify which assertions were failing, but I could make them more or less verbose depending on what you want.